doc/dev/perf_counters.rst

```diff
@@ -4,6 +4,8 @@
  Perf counters
 ===============
 
+.. note:: Perf counters were introduced in the Reef release of Ceph.
```

Comment:
This is false -- perf counters were introduced in 2011-2012 and were present in argonaut (the first stable release).  What was introduced in reef are labeled perf counters.

This should be moved to Labeled Perf Counters section below.

Reply:
I'll do it in a few hours, when I wake up. 

Thanks for the correction, @idryomov.

Reply:
https://github.com/ceph/ceph/pull/57886